docs/Changelog.md

```diff
@@ -47,6 +47,7 @@
 * [#3253](https://github.com/TouK/nussknacker/pull/3253) Separate validation step during scenario deployment
 * [#3328](https://github.com/TouK/nussknacker/pull/3328) Schema type aware serialization of `NkSerializableParsedSchema`
 * [#3071](https://github.com/TouK/nussknacker/pull/3071) More strict avro schema validation: include optional fields validation, reduce number of validation modes.
+* [#3287](https://github.com/TouK/nussknacker/pull/3287) Add low level kafka components by feature flag
```

Comment:
Can you move this changelog entry to grouped entry: Universal kafka source/sink ... ? IMO it will be easier for users if you just change this:
> Universal kafka source/sink, handling multiple scenarios like: avro message for avro schema, json message for json schema 

to:
> Universal kafka source/sink, handling multiple scenarios like: avro message for avro schema, json message for json schema. Legacy, low level kafka components can be turned off by new XXX flag

Reply:
done

Reply:
done